src/CameraOnvif.cpp

```diff
@@ -187,9 +200,12 @@ void CameraOnvif::reportError(){
 
 void CameraOnvif::setCredentials(){
     soap_wsse_delete_Security(m_private->soap);
-    if (soap_wsse_add_Timestamp(m_private->soap, "Time", 10)
-        || soap_wsse_add_UsernameTokenDigest(m_private->soap, "Auth", m_user.c_str(), m_pass.c_str()))
+    bool error1 = soap_wsse_add_Timestamp(m_private->soap, "Time", 10);
```

Comment:
Shouldn't that be the same timeout than the rest ?

Reply:
This bug came from tutorial :/